content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/_cq_dialog/.content.xml

```diff
@@ -95,6 +95,18 @@
                                                         value="h6"/>
                                                 </items>
                                             </defaulttypes>
+                                            <linkURL
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="cq/gui/components/coral/common/form/pagefield"
+                                                fieldDescription="Make the title a link to another resource."
```

Comment:
@Shivanshmathur - we could be more specific about what we accept here. 

"Links the title. Path to a content page, external URL or page anchor."

Reply:
Done.